docs/releases/1.10.txt

```diff
@@ -383,6 +389,9 @@ Models
 * A proxy model may now inherit multiple proxy models that share a common
   non-abstract parent class.
 
+* ``Model.__init__`` now recognizes keyword arguments setting values of
```

Comment:
Again, since there is no official support for virtual fields, I'm not sure if this is important to mention. I don't really know what it means though.